website/common/script/ops/releaseBoth.js

```diff
@@ -44,21 +46,30 @@ module.exports = function releaseBoth (user, req = {}, analytics) {
   for (animal in content.pets) {
     if (user.items.pets[animal] === -1) {
       giveTriadBingo = false;
+    } else if (user.items.pets[animal] === 0 || user.items.pets[animal] === null) {
```

Comment:
Hi, I'm not 100% sure, maybe @paglias or someone else would have some input on this? My thoughts are that we shouldn't include undefined, because we only want to check the pets that exist within the user.items.pets object. If a pet is missing from the object then it'd fail if we checked for undefined. Also, the only valid values should be -1, 0, null or positive int. 

Lastly, I did try it before using else if(!user.items.pets[animal]) - which checks for null, 0 or undefined - but this ended up failing previously written unit tests for releaseBoth.js - which is also another reason I didn't check for undefined